microsoft-edge/webview2/how-to/publish-uwp-app-store.md

```diff
@@ -0,0 +1,139 @@
+﻿---
+title: Publish a UWP WebView2 app to the Microsoft Store
+description: How to publish a Universal Windows Platform (UWP) app that hosts the WebView2 control to the Microsoft Store.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 04/18/2022
+---
+# Publish a UWP WebView2 app to the Microsoft Store
+
+A _UWP WebView2 app_ is a Universal Windows Platform (UWP) app that hosts the WebView2 control.  To make a UWP WebView2 app widely available, you can publish it to the Microsoft Store, by following the steps below.
+
+For general background information about publishing to the Windows Store, see [Publish Windows apps and games](/windows/uwp/publish/), at Windows App Development documentation > UWP.  The steps below provide the specific details about how to publish a UWP WebView2 app.
+
+
+<!-- ====================================================================== -->
+## Step 1: Configure a Microsoft Store account in Visual Studio
+
+*  Make sure you have a Microsoft Store account configured in Microsoft Visual Studio.  See [Get registered](https://developer.microsoft.com/microsoft-store/register/), about developer accounts for the Microsoft Store.
+
+Then continue with the steps below.
+
+
+<!-- ====================================================================== -->
+## Step 2: Test the non-packaged app
+
+<!-- Why test these particular aspects?  What other aspects?  What's the focus here? -->
```

Comment:
@davidshoe - Why test these particular aspects?  What other aspects?  What's the focus here?

Aiming for a good Sentence 1 for each h2 section.

Reply:
Ensure base functionality to save time later so you don't get bounced out of the store ingestion.